examples/nextjs-blog/pages/authors/list.js

```diff
@@ -1,13 +1,50 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
 import { flamelinkApp as app } from '../../utils/flamelink'
 import { Link } from '../../routes'
 
-const Authors = function(props = {}) {
-  const { authors } = props
-  return Object.values(authors).map(author => (
-    <h3 key={author.id}>
-      <Link route={`/authors/${author.id}`}>{author.displayName}</Link>
-    </h3>
-  ))
+class Authors extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {
+      authors: null,
+      loading: true,
+    }
+  }
+
+  UNSAFE_componentWillMount() {
```

Comment:
think you can safely use componentDidMount (I think it is actually preferred for client side data fetching)